int32 id
float64 angle
int32 load
int32 voltage
int32 temperature
